asm-lsp/config_builder.rs

```diff
@@ -189,7 +189,11 @@ fn prompt_project_path(opts: &GenerateOpts) -> PathBuf {
             fallback_enter(&mut true_path);
             return true_path;
         };
-        for entry in dir_reader.filter_map(std::result::Result::ok) {
+        for entry in dir_reader.filter_map(|entry| {
+            entry
+                .ok()
+                .and_then(|entry| entry.path().is_dir().then_some(entry))
```

Comment:
I don't think we want to just show directories here, since the project configs allow for paths to directories as well as paths to specific files. It does make sense that directories would be the much more common case, though. I could add some sorting logic to make the directories always display before files if that would be helpful.

Reply:
Let me see if I can find a moment to make the logic sort and bring directories first. :)